app/decorators/proxy_rule_decorator.rb

```diff
@@ -0,0 +1,15 @@
+class ProxyRuleDecorator < ApplicationDecorator
+
+  self.include_root_in_json = false
+
+  def pattern
+    pattern_value = super
+    backend_api_path ? File.join("/#{backend_api_path}", pattern_value) : pattern_value
+  end
+
+  private
+
+  def backend_api_path
```

Comment:
Does it really matter to memoize a value in a Hash?

Reply:
not that much..